include/boost/url/detail/impl/params_encoded_iterator_impl.ipp

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+// Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_DETAIL_IMPL_PARAMS_ENCODED_ITERATOR_IMPL_IPP
+#define BOOST_URL_DETAIL_IMPL_PARAMS_ENCODED_ITERATOR_IMPL_IPP
```

Comment:
the macro name again doesn't match the file...

Reply:
What should the macro be?

`boost/url/detail/impl/params_encoded_iterator_impl.ipp`
`BOOST_URL_DETAIL_IMPL_PARAMS_ENCODED_ITERATOR_IMPL_IPP`

Reply:
hmm you're right